static/app/views/insights/common/components/widgets/overviewJobsChartWidget.tsx

```diff
@@ -45,37 +45,54 @@ export default function OverviewJobsChartWidget(props: LoadableChartWidgetProps)
 
   const fullQuery = `span.op:queue.process ${query}`.trim();
 
-  const {data, isLoading, error} = useSpanSeries(
+  const {data, isLoading, error} = useFetchSpanTimeSeries(
     {
       ...pageFilterChartParams,
-      search: fullQuery,
+      query: fullQuery,
       yAxis: ['trace_status_rate(internal_error)', 'count(span.duration)'],
-      referrer: Referrer.JOBS_CHART,
+      pageFilters: props.pageFilters,
     },
-    Referrer.JOBS_CHART,
-    props.pageFilters
+    Referrer.JOBS_CHART
   );
 
-  const plottables = useMemo(() => {
-    return [
-      new Bars(convertSeriesToTimeseries(data['count(span.duration)']), {
+  const countPlottable = useMemo(() => {
+    const timeSeries = data?.timeSeries || [];
+    const countSeries = timeSeries.find(ts => ts.yAxis === 'count(span.duration)');
```

Comment:
Curious, can we just assume that the `count(span.duration)` series will always be the second one, because it's the second `yAxis` provided? Regardless, explicitly checking against the yAxis seems more robust.